syntax = "proto3";
option go_package = "github.com/harlow/go-micro-services/services/search/proto";

package search;

// Search service returns best hotel choices for a user.
service Search {
  rpc Nearby(NearbyRequest) returns (SearchResult);
}

message NearbyRequest {
  float lat = 1;
  float lon = 2;
  string inDate = 3;
  string outDate = 4;
}

message SearchResult {
  repeated string hotelIds = 1;
}
